app/src/main/java/com/keylesspalace/tusky/interfaces/StatusActionListener.java

```diff
@@ -37,4 +37,17 @@
      * @param position    The position of the status in the list.
      */
     void onContentCollapsedChange(boolean isCollapsed, int position);
+
+    /**
+     * called when the reblog count has been clicked
+     * @param position The position of the status in the list.
+     */
+    default void onShowReblogs(int position) {}
```

Comment:
It's the general interface for all status containers but you can only to these things on one of them so maybe it should be a separate interface which extends the other one, idk. Maybe later.

Reply:
yes I know, but the `StatusDetailedViewHolder`  `StatusViewHolder` class hierachy makes it difficult to pass different listeners, I think the default interface methods are the best tradeoff